server/py/framework/db/init_db.py

```diff
@@ -15,8 +15,17 @@
 
 from mlrun.common.db.sql_session import get_engine
 
-from framework.db.sqldb.models import Base
+from framework.db.sqldb.models import AlertActivation, Base
 
 
 def init_db() -> None:
-    Base.metadata.create_all(bind=get_engine())
+    engine = get_engine()
+    if engine.name == "sqlite":
+        tables_to_create = [
+            table
+            for table in Base.metadata.tables.values()
+            if table.name != AlertActivation.__tablename__
+        ]
+        Base.metadata.create_all(bind=engine, tables=tables_to_create)
+    else:
+        Base.metadata.create_all(bind=engine)
```

Comment:
1. How does this allow running unit tests if we don't create the tables at all?
2. a more extendable version will be something like a function `get_sqlite_tables` that resolves the list. It will keep the `init_db` function leaner.

Reply:
@TomerShor  the problem is that:

1. `SQLIte` doesn't support autoincrement on composite PK
2. our code considers that `id` is autoincrementable thus we don't generate id

as a result, it doens't make sense even to create alert_activations table within tests as we can do nothing with table. Before this PR, i tried to make a trick with no putting `autoincrememt`. But this approach is quite hacky for the real code itself. So I decided that it's better to affect the tests rather than the actual code. All the tests for partitioned table should be integration/system until we decide to move to mysql for unit tests.

Reply:
I tend to agree with @TomerShor in that I'd maintain in the `models.py` file a method called `get_partitioned_table_names()` or something - just so it's co-located with the tables themselves. This code will check that the table name is not in the returned list. This way, a developer that makes another partitioned table only needs to update that function, and doesn't need to be aware that this code here also needs to change.

Reply:
@theSaarco Added the function and updated CONTRIBUTION.md with instructions.